.expeditor/update_version.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+#
+# After a PR merge, Chef Expeditor will bump the PATCH version in the VERSION file.
+# It then executes this file to update any other files/components with that new version.
+#
+
+set -evx
+
+sed -i -r "s/^(\s*)VERSION = \".+\"/\1VERSION = \"$(cat VERSION)\"/" lib/knife_ec_backup/version.rb
```

Comment:
`lib/knife_ec_backup_version.rb` has single quotes around the version, so you need this instead:

```
sed -i -r "s/^(\s*)VERSION = '.+'/\1VERSION = '$(cat VERSION)'/" lib/knife_ec_backup/version.rb
```

Reply:
(chefstyle wants double quotes around version, but let's not worry about making chefstyle happy in this PR.)

Reply:
might as well put a comment in `lib/knife_ec_backup/version.rb` that says when you do change these to double quotes to also update `.expeditor/update_version.sh`.